packages/esm-appointments-app/translations/am.json

```diff
@@ -104,7 +104,6 @@
   "queueAddedSuccessfully": "",
   "queueEntryError": "Error adding patient to the queue",
   "reasonForChanges": "Reason For Changes",
-  "rescheduled": "Rescheduled",
```

Comment:
All these changed got auto-committed when I tried to push, not sure exactly how this is supposed to work, but I can clean up on merge?